sympy/matrices/tests/test_matrices.py

```diff
@@ -196,20 +196,20 @@ def test_power():
     assert Matrix([[1, 2], [3, 4]])**Integer(2) == Matrix([[7, 10], [15, 22]])
 
     A = Matrix([[33, 24], [48, 57]])
-    assert (A**(S(1)/2))[:] == [5, 2, 4, 7]
+    assert ( (A**(S(1)/2)).doit() )[:] == [5, 2, 4, 7]
     A = Matrix([[0, 4], [-1, 5]])
-    assert (A**(S(1)/2))**2 == A
+    assert ((A**(S(1)/2))**2).doit() == A
```

Comment:
Should I change the test to : `assert (A ** (S(1)/2)).doit() ** 2 == A `?

Because clubbing together the exponents would not work everytime ::
( (-1) ^ 2 )^(1/2) not equal to (-1)^(2*(1/2)).(Reference :: [Symplification](http://docs.sympy.org/latest/tutorial/simplification.html))